src/components/measureGroups/MeasureObservationDetails.tsx

```diff
@@ -69,75 +70,103 @@ const MeasureObservationDetails = ({
             Remove
           </DSLink>
         )}
-        <Select
-          placeHolder={{ name: "-", value: "" }}
-          required={required}
-          readOnly={!canEdit}
-          label={label ? label : "Observation"}
-          id={`measure-observation-${name}`}
-          data-testid={`select-measure-observation-${name}`}
-          inputProps={{
-            "data-testid": `measure-observation-${name}-input`,
-          }}
-          style={{ width: 300 }}
-          value={measureObservation?.definition || ""}
-          onChange={(e) => {
-            if (onChange) {
-              onChange({
-                ...measureObservation,
-                definition: e.target.value,
-              });
-            }
-          }}
-          options={[
-            !required && (
-              <MenuItem key="-" value="">
-                -
-              </MenuItem>
-            ),
-            ...cqlFunctionNames.map((o) => (
-              <MenuItem key={o} value={o}>
-                {o}
-              </MenuItem>
-            )),
-          ]}
-        />
+        {canEdit && (
+          <Select
+            placeHolder={{ name: "-", value: "" }}
+            required={required}
+            disabled={!canEdit}
+            label={label ? label : "Observation"}
+            id={`measure-observation-${name}`}
+            data-testid={`select-measure-observation-${name}`}
+            inputProps={{
+              "data-testid": `measure-observation-${name}-input`,
+            }}
+            style={{ width: 300 }}
+            value={measureObservation?.definition || ""}
+            onChange={(e) => {
+              if (onChange) {
+                onChange({
+                  ...measureObservation,
+                  definition: e.target.value,
+                });
+              }
+            }}
+            options={[
+              !required && (
+                <MenuItem key="-" value="">
+                  -
+                </MenuItem>
+              ),
+              ...cqlFunctionNames.map((o) => (
+                <MenuItem key={o} value={o}>
+                  {o}
+                </MenuItem>
+              )),
+            ]}
+          />
+        )}
+        {!canEdit && (
+          <div>
+            <AutoComplete
```

Comment:
Is there a reason for using an Autocomplete just to disable it? Seems like it's just being used as a label basically?